pkg/event_processor/testdata/952282712204824.bin

```diff
@@ -2,7 +2,7 @@ HTTP/1.1 200 OK
 Server: nginx/1.14.0 (Ubuntu)
 Date: Wed, 29 Jun 2022 09:44:35 GMT
 Content-Type: text/html; charset=UTF-8
-Content-Length: 74332
+Content-Length: 20124
```

Comment:
Why has the number of bytes increased here?

Reply:
Actually, the number of bytes decreased here (from 74332 -> 20124). 

The total response body length for the following `*.bin` file is obviously less than 74332 which will cause the error `DumpError: Unexpected EOF`. This problem has existed before and could be verified through https://github.com/gojue/ecapture/actions/runs/8094294516/job/22118690803#step:8:226
    
```json
{"DataType":0,"Timestamp":952282712204824,"Pid":469953,"Tid":469953,"DataLen":4096,"Comm":[99,117,114,108,0,0,0,0,0,0,0,0,0,0,0,0],"Fd":5,"Version":771}
{"DataType":0,"Timestamp":952282715334491,"Pid":469953,"Tid":469953,"DataLen":4096,"Comm":[99,117,114,108,0,0,0,0,0,0,0,0,0,0,0,0],"Fd":5,"Version":771}
{"DataType":0,"Timestamp":952282731469385,"Pid":469953,"Tid":469953,"DataLen":4096,"Comm":[99,117,114,108,0,0,0,0,0,0,0,0,0,0,0,0],"Fd":5,"Version":771}
{"DataType":0,"Timestamp":952282736644923,"Pid":469953,"Tid":469953,"DataLen":4096,"Comm":[99,117,114,108,0,0,0,0,0,0,0,0,0,0,0,0],"Fd":5,"Version":771}
{"DataType":0,"Timestamp":952282737583175,"Pid":469953,"Tid":469953,"DataLen":4096,"Comm":[99,117,114,108,0,0,0,0,0,0,0,0,0,0,0,0],"Fd":5,"Version":771}
```